scrapers/scraper.py

```
@@ -0,0 +1,424 @@
+            if "The_Great_Inferno" in url:
+                index = 4
+            else:
+                index = 2
+        else:
+            index = 0
+
+        tables = soup.find_all('table')
+        for item in tables[index].find_all('tr')[2:]:
+            try:
+                area = list(item.find_all('td')[2].children)[0]['href'].replace('/wiki/', '')
+            except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)